.github/workflows/build_publish_docker.yml

```diff
@@ -18,15 +18,15 @@ jobs:
     name: Docker ${{ matrix.container }} on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
-        fail-fast: false
-        matrix:
-            os: ["ubuntu-latest"]
-            container: ["detector",
-                        "inspector",
-                        "logcollector",
-                        "logserver",
-                        "prefilter"
-                        ]
+      fail-fast: false
```

Comment:
Why got all files formatted? Do you use black?

Reply:
See above conversation